grakel_replace/optimize.py

```diff
@@ -81,21 +116,34 @@ def optimize_acqf_graph(
         raise ValueError("train_graphs cannot be None.")
 
     sampled_graphs = sample_graphs(train_graphs, num_samples=num_graph_samples)
+    gp = acq_function.model
+    covar_module = gp.covar_module
 
     best_candidates, best_scores = [], []
 
+    TODO_GRAPH_COLUMN_INDEX = bounds.shape[1] - 1
```

Comment:
... we need to know which column the graph is in for the `fixed_features` below. Probably better to lift this logic out of this function into the caller.